packages/govuk-frontend/src/govuk/components/button/button.yaml

```
@@ -145,25 +145,29 @@ examples:
       classes: govuk-button--warning
   - name: Inverse
     description: A button that appears on dark backgrounds
+    invertLayoutColours: true
     data:
       name: Inverse
       text: Inverse button
       classes: govuk-button--inverse
   - name: Inverse disabled
+    invertLayoutColours: true
     data:
       name: Inverse
       text: Inverse button disabled
       classes: govuk-button--inverse
       disabled: true
   - name: Inverse link
     description: A link button for actions that appear on dark backgrounds
+    invertLayoutColours: true
     data:
       name: Inverse
       text: Inverse button
       href: '/'
       classes: govuk-button--inverse
   - name: Inverse start button
     description: A start button that appears on dark backgrounds
+    invertLayoutColours: true
```

We already use `previewLayout: full-width`

How about `previewLayoutInverse: true`?

```suggestion
    previewLayoutInverse: true
```

---

I opted against using `inverse` on the basis that it might not be descriptive enough ("Inverse what? The component? The text direction?"), but `previewLayoutInverse` sounds like a good candidate.

I'm also pondering whether using the word 'dark' might be better than 'inverse' as that (probably?) better describes what it is—it's the component that is inverted, rather than the layout. Will have a ponder.

---

Yeah, all valid points but worked for `--inverse` 🤷‍♂️

---

Or another idea?

```suggestion
    previewLayoutModifiers:
      - inverse
```